lib/griddler/mailgun/adapter.rb

```diff
@@ -79,6 +80,14 @@ def param_or_header(key)
         end
       end
 
+      def content_id_map
+        begin
+          JSON.parse(params['content-id-map'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.